experimental/orchestrator/README.md

```diff
@@ -97,7 +97,7 @@ const recoResult = await dispatchRecognizer.recognize(context);
 **C#:**
 - Add reference to `Microsoft.Bot.Builder.AI.Orchestrator` package.
 - Set your project to target `x64` platform
-- Install latest supported version of [Visual C++](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads)
+- Install latest supported version of [Visual C++ redistributable package](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads)
```

Comment:
This is a dupe of line 58.  Do we need to mention it again?

Reply:
Agree. Will remove